src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/NoLineWrap/InputnolinewrapGood.java

```diff
@@ -0,0 +1,18 @@
+package com.puppycrawl.tools.checkstyle.checks.whitespace.NoLineWrap;
+
+/**
+ *
+ * @author divsidhu-26
+ */
+import com.puppycrawl.tools.checkstyle.TreeWalker;
+
+import javax.accessibility.AccessibleAttributeSequence;
+import java.util.concurrent.atomic.AtomicIntegerFieldUpdater;
```

Comment:
@divsidhu-26 Inputs don't go in the `main` folder. They go in the `test` folder under `resources`.